Dockerfile

```diff
@@ -245,9 +245,21 @@ USER buildbot
 RUN /bin/bash -c 'phpbrew init && source ~/.phpbrew/bashrc && phpbrew install 5.6 +default && \
     phpbrew app get composer'
 
+################################################################################
+#
+# Emacs Cask
+#
+################################################################################
 USER root
+RUN add-apt-repository ppa:kelleyk/emacs
```

Comment:
yeah, v24.4 would work just fine, but distro version is 24.3, which is too old and useless, even some common package like `scala-mode` or `clojure-mode` (for code highlight in org export) will not work.

to get 24.4 or above IMO there're 3 options:
1. ppa
2. build from source code(there's already [build-essentials](https://github.com/netlify/build-image/blob/master/Dockerfile#L12) installed, so I guess it would just work)
3. change the base image from Ubuntu 14 to 16, which have emacs 24.5 included(but could be big impact to other packages in build-image)